src/Cli/Microsoft.TemplateEngine.Cli/Commands/create/TemplateCommand.cs

```diff
@@ -213,7 +213,7 @@ internal async Task<NewCommandStatus> InvokeAsync(ParseResult parseResult, Cance
                 }
             }
 
-            return instantiateTask.Result;
+            return Task.Run(() => instantiateTask).Result;
```

Comment:
Similar to the comment on `InstantiateCommand.TabCompletion.cs`, the task here was already awaited on line 178. I don't believe this `Task.Run` does anything.